Chart.yaml

```diff
@@ -6,5 +6,5 @@ name: terraform-enterprise
 kubeVersion: ">=1.21.0-0"
 description: Official HashiCorp Terraform-Enterprise Chart
 type: application
-version: 1.3.1
-appVersion: "v202408-1"
+version: 1.3.2
+appVersion: "v202409-1"
```

Comment:
💭 I wonder whether we should remove the appVersion altogether. 

The [definition of appVersion](https://helm.sh/docs/topics/charts/) according to helm docs is: 

> appVersion: The version of the app that this contains (optional). Needn't be SemVer. Quotes recommended.

I then want to check which version the helm chart contains, and I see that it contains no version at all: 

https://github.com/hashicorp/terraform-enterprise-helm/blob/8fae102b101906c8140818acc826f1a1d7640377/values.yaml#L18

I recommend to either remove the appVersion, or to actually use it, i.e. set this release as the default version. 
Does that make sense?